### PR TITLE
impr(quote): add 3 spanish quotes (KANGRIZ)

### PR DESCRIPTION
**Pull request by KANGRIZ**
One of the texts is long, and the other two are short.  Id: 142, 143 and 144

### Description

Closes #

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/5991687204

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/5991774274

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/5991796242

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/5991823960

**Comment by KANGRIZ** [comment]
three Spanish texts were added to the spanish.json document. In addition to some changes in spelling and punctuation.

File: frontend/static/quotes/spanish.json [hunk 1]
```diff
@@ -840,6 +840,24 @@
       "source": "Pablo Escobar",
       "length": 70,
       "id": 141
-    }
+    },
+    {
+      "text": "Existen otros centenares de ellos tan pequeños a veces, que es difícil distinguirlos aún con la ayuda del telescopio. Cuando un astrónomo descubre uno de estos planetas, le da por nombre un número. Le llama, por ejemplo, \"el asteroide 3251\". Tengo poderosas razones para creer que el planeta del cual venía el principito era el asteroide B612. Este asteroide ha sido visto solo una vez con el telescopio en 1909, por un astrónomo turco. Este astrónomo hizo una gran demostración de su descubrimiento en un congreso Internacional de Astronomía. Pero nadie le creyó a causa de su manera de vestir. Las personas mayores son así. Felizmente, para la reputación del asteroide B 612, un dictador turco impuso a su pueblo, bajo pena de muerte, el vestido a la europea. Entonces el astrónomo volvió a dar cuenta de su descubrimiento en 1920 y como lucía un traje muy elegante, todo el mundo aceptó su demostración",
```
Comment (Miodec) [inline frontend/static/quotes/spanish.json:845]: Missing period at the end of the quote.

**Review by Miodec — CHANGES_REQUESTED** [review]
Please leave a comment with translations so I can verify their content.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6005623272

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6054890531

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6054947115

**Comment by KANGRIZ** [comment]
Hi!

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6054961831

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6054996821

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6055076776

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6055116580

**Pull request closed without merging**